.github/workflows/push-package-solarwinds.yml

```diff
@@ -0,0 +1,37 @@
+# Copyright (c) 2023 SolarWinds, LLC.
```

Comment:
Is there a reason for having this header instead of the full Apache license? Same for the other file.

Reply:
No particular reason. Just realized there is no header for all workflows so I removed it. Other files are out of my hand since they are from upstream.